icu4c/source/test/fuzzer/break_iterator_fuzzer.cc

```diff
@@ -11,8 +11,8 @@ IcuEnvironment* env = new IcuEnvironment();
 
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
   UErrorCode status = U_ZERO_ERROR;
-  icu::UnicodeString str(UnicodeStringFromUtf8(data, size));
-
+  icu::UnicodeString fuzzstr(reinterpret_cast<const UChar*>(data),
+                             static_cast<int>(size) / sizeof(UChar));
```

Comment:
Just to exactly match the way ICU delcares the UnicodeString() constructor,
fuzzstr(reinterpret_cast<const char16_t *>(data),
 static_cast<int32_t>(size) / sizeof(char16_t));

Similar for other places.